docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -159,7 +159,7 @@ rm -f id*
 cd -
```

Comment:
Changes here LGTM but also CCing @gibizer @bogdando .

Reply:
Look OK to me too